Resources/Prototypes/_DeadSpace/Entities/Markers/LightPoint/light_points.yml

```
@@ -0,0 +1,17 @@
+- type: entity
+  id: MarkerLightSource
+  name: источник света
+  description: и сказал админ "да будет свет!"
+  suffix: Включите ВВ (панель песочницы)
```

```suggestion
  suffix: Спавнер
```
Включение ВВ его не отображает